user-guide/settings/connections.md

```diff
@@ -9,29 +9,22 @@ position: 20
 
 # Connections Menu
 
-The __Connections__ menu exposes some finer control over proxy settings and remote connection. 
+The __Connections__ menu exposes options that allow for granular control over proxy settings and remote connection.
```

Comment:
Some of the minor suggestions for this article feel more like nitpicking, sorry.